src/models/slices/directanswer.ts

```diff
@@ -1,5 +1,11 @@
 import { FeaturedSnippetDirectAnswer, FieldValueDirectAnswer } from '@yext/answers-core';
 
+/**
+ * Maintains the direct answer associated with the latest search.
```

Comment:
Yes, we should use them because that is our convention in Core. If I recall, they are inferred, but API extractor gives a warning if you don't include them

Reply:
I thought this might be good to do as part of the API extractor item if we needed to work out what should be `@public` and not. Unless, everything that is currently being exported should be `@public`, then I can just add that now as I add the docs.

Reply:
That makes sense. There will almost certainly need to be some changes once we add API extractor, and it will tell us exactly what those changes need to be